components/navigation/navigation.css

```diff
@@ -0,0 +1,61 @@
+.nav-boxx {
+    height: 5rem;
+    width: 80%;
+    /* border: 1px solid black; */
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    box-shadow: 0px 4px 12px -3px var(--text-color);
+    background-color: var(--primary-color);
+}
+
+.logo {
+    font-size: 2rem;
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+    margin-left: 1rem;
+    font-family: cursive;
+    color: var(--secondary-color);
+    text-shadow: 2px 3px var(--text-color);
+}
+
+.action-boxx {
+    display: flex;
+    flex-direction: row;
```

Comment:
Again here also you could avoid `flex-direction: row` as it is a default value, so it would work without explicitly specifying it.